src/clients/rest/rest_client.ts

```diff
@@ -31,8 +31,14 @@ class RestClient extends HttpClient {
 
     const link = ret.headers.get('link');
     if (params.query && link !== undefined) {
+      const defaultLimit = '50';
```

Comment:
I might put this as a static variable on the class instead, as we have with the regex: 

```suggestion
     private static DEFAULT_LIMIT = '50';
```

and then we can still reference it with the same type of conditional statement.

Reply:
+